### ISSUE TITLE
Inference

### ISSUE THREAD

**Issue by SilasAntonisen** [guxm2021/ALT_SpeechBrain#3]
Hello,
I find your work very interesting and would very much like to experiment with it, however i have some difficulty figuring out how to use the model for inference. I would like to use an ALT system to transcribe my singing from audio files and thought this model might be a really good solution. Would it be possible to get some guidelines for how to do so?
Thanks

**Comment by SilasAntonisen** [comment]
In case anyone is wondering, i simply transcribe my own audio files by commenting out brain.fit in the training script such that i directly generate a wer.txt. There are obviously no ground truth labels for my own recordings, but i am not interrested in the wer, only the generated transcript. I achieve this by assigning a random ground truth label and then simply read the predictions from the wer.txt. There are probably better ways to do this by making an inference class or deataching the WER estimation, but this is just a quick and dirty approach i though i would share.
